content/en/docs/languages/java/sdk.md

```diff
@@ -0,0 +1,1382 @@
+---
+title: Manage Telemetry with SDK
+linkTitle: Manage Telemetry with SDK
```

Comment:
A `linkTitle` is meant to be short. How about:

```suggestion
linkTitle: SDK
```

Reply:
While "Manage Telemetry with SDK" is very long, I don't like this being shortened to "SDK", since endusers will not understand what to expect from clicking that in the navigation, see my [discussion comment](https://github.com/open-telemetry/opentelemetry.io/discussions/4853#discussioncomment-10212657) on that topic as well. 

The problem is that we need both (the manage part and the "SDK") to be clear what will behind that link in the navigation. Personally, I would prefer "Manage Telemetry" over "SDK", but I can imagine that people think otherwise.

Reply:
> Personally, I would prefer "Manage Telemetry" over "SDK", but I can imagine that people think otherwise.

Yeah I think we need both. We need the "SDK" for users who are already familiar with opentelemetry concepts, since it isn't obvious that "manage telemetry" is SDK content. We need "manage telemetry" for users who are new to opentelemetry and don't have intuition about what the sdk is.

One consideration is that we discussed adding a "Learn the Basics" / "Introduction" section, which will explain key concepts. If every new user first reads that content, then they'll learn that the sdk is used to managed telemetry, and we might be able to get away with omitting the "manage telemetry" portion of the title. The problem is not all users will read the "Learn the basics" / "Introduction" content. To what extent to we want to accommodate users who consume information out of order?

Reply:
Moving this out into a dedicated discussion: https://github.com/open-telemetry/opentelemetry.io/discussions/5039

Reply:
Marking as resolved.